assets/js/settings/blocks/store-api-nonce.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import { getSetting } from '../shared';
+
+/**
+ * Returns whether or not this is a non GET wc/store API request.
+ *
+ * @param {Object} options Options given to apiFetch.
+ *
+ * @return boolean
+ */
+const isStoreApiGetRequest = ( options ) => {
+	const url = options.url || options.path;
+	if ( ! url || ! options.method || options.method === 'GET' ) {
+		return false;
+	}
+	return /wc\/store\//.exec( url ) !== null;
+};
+
+/** cache for nonce initialized from hydration  */
```

Comment:
I know this is my example, but we should probably just use `//` for the comment block (and add a full stop at the end of the comment sentence) here.